content/blog/tidymodels-2023-q3/index.Rmd

```diff
@@ -0,0 +1,175 @@
+```
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles.
+
+Since the beginning of 2021, we have been publishing [quarterly updates](https://www.tidyverse.org/categories/roundup/) here on the tidyverse blog summarizing what's new in the tidymodels ecosystem. The purpose of these regular posts is to share useful new features and any updates you may have missed. You can check out the [`tidymodels` tag](https://www.tidyverse.org/tags/tidymodels/) to find all tidymodels blog posts here, including our roundup posts as well as those that are more focused, like this post from the past couple of months:
+
+-   [New interface to validation splits](https://www.tidyverse.org/blog/2023/08/validation-split-as-3-way-split/)
+
+Since [our last roundup post](https://www.tidyverse.org/blog/2022/12/tidymodels-2022-q4/), there have been CRAN releases of `r nrow(tm_data)` tidymodels packages. Here are links to their NEWS files:
+
+```{r}
+#| echo: FALSE
+#| results: asis
+
+cat(tm_data$news, sep = "\n")
+```
+
+We'll highlight a few especially notable changes below: Updated workshop material, new K-means engines and quality of life improvements in rsample. First, loading the collection of packages:
+
+```{r}
+#| message: FALSE
+library(tidymodels)
+library(tidyclust)
+
+data("ames", package = "modeldata")
+```
+
+## Workshops
+
+One of the biggest areas of work for our team this quarter was getting ready for this year's [posit::conf](https://posit.co/conference/). This year, two 1-day workshops were available; "Introduction to tidymodels" and "Advanced tidymodels". All the material can be found on our workshop website
+[workshops.tidymodels.org](https://workshops.tidymodels.org/), with this workshop being archived as [posit::conf 2023 workshop](https://workshops.tidymodels.org/archive/2023-09-posit-conf/).
+
+Unless otherwise noted (i.e. not an original creation and reused from another source), these educational materials are licensed under Creative Commons Attribution [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## Tidyclust update
+
+The latest release of tidyclust featured a round of bug fixes, documentation improvements and quality-of-life improvements.
+
+This release adds 2 new engines to the `k_means()` model. [klaR](https://tidyclust.tidymodels.org/reference/details_k_means_klaR.html) to run K-Modes models and [clustMixType](https://tidyclust.tidymodels.org/reference/details_k_means_clustMixType.html) to run K-prototypes. K-Modes is the categorical analog to K-means, meaning that it is intended to be used on only categorical data, and K-prototypes is the more general method that works with categorical and numeric data at the same time.
+
+If we were to fit a K-means model to a mixed-type data set such `ames`, it would work but under the hood, it applied a dummy transformation on the categorical predictors.
```

Comment:
```suggestion
If we were to fit a K-means model to a mixed-type data set such as `ames`, it would work, but under the hood, the model would apply a dummy transformation on the categorical predictors.
```